test/unit/ssz_ex_test.exs

```diff
@@ -133,4 +133,52 @@ defmodule Unit.SSZExTest do
 
     assert_roundtrip(serialized, sync, SszTypes.SyncCommittee)
   end
+
+  test "serialize and deserialize bitlist" do
```

Comment:
In this tests you're only adding bytestrings (number of bits multiple of 8). We should add some bitstrings too, for instance <<10::5>>

Reply:
:sweat_smile: this test breaks everything

Reply:
I think these kinds of cases should return an error because getting the bitlist's length is impossible. 
The operation  to get the length is `byte_index (in this case 0) * 8 + 7 - leading_zeros (in this case 1) -> 6`

Reply:
I think that valid encoded `bitlists` are always multiple of 8 :thinking: